config/locales/en.yml

```diff
@@ -4,6 +4,8 @@ en:
     dashboard_welcome:
       welcome: "Welcome to Active Admin. This is the default dashboard page."
       call_to_action: "To add dashboard sections, checkout 'app/admin/dashboard.rb'"
+    admin_user:
+      details: "Admin Details"
```

Comment:
I think I'd go with a different key name. Otherwise it could be interpreted that there's a `details` column on the `AdminUser` model.